data/webbrowser.yaml

```diff
@@ -799,6 +799,52 @@ sources:
 supported_os: [Darwin, Linux, Windows]
 urls: ['https://forensicswiki.xyz/wiki/index.php?title=Google_Chrome#Configuration']
 ---
+name: ChromeSessions
```

Comment:
ChromeSessions => ChromeSessionsAndTabs

Reply:
This definition also seems to overlap with ChromeSessionStorage

Reply:
Fixed a typo - there shouldn't be any overlap with the ChromeSessionStorage definitions now.